flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/ddl/SqlCreateMaterializedTable.java

```diff
@@ -102,7 +101,7 @@ public SqlCreateMaterializedTable(
 
     @Override
     public SqlOperator getOperator() {
-        return OPERATOR;
+        return CREATE_OPERATOR;
     }
```

Comment:
I guess if you drop this method override it will return the operator which is passed to the constructor and this is what we want, right?